cnxarchive/views/helpers.py

```diff
@@ -66,6 +66,14 @@ def get_content_metadata(id, version, cursor):
         #       Until then we will do the replacement here.
         result['mediaType'] = portaltype_to_mimetype(result['mediaType'])
 
+        cursor.execute("""
```

Comment:
Rather than a second fetch here, let's do a linked change over in cnx-db, to return baked status in `get-module-metadata`